persist-cli/src/main/java/io/ballerina/persist/nodegenerator/syntax/clients/RedisClientSyntax.java

```diff
@@ -332,15 +332,35 @@ private static String getJoinMetaData(Entity entity) {
             if (entityField.getRelation() != null) {
                 String relationType = BalSyntaxConstants.REDIS_ONE_TO_ONE;
                 Entity associatedEntity = entityField.getRelation().getAssocEntity();
+//                List<String> references = entityField.getRelation().getReferences();
```

Comment:
Is this line necessary?

Reply:
No, we can remove that. I have removed it in https://github.com/ballerina-platform/persist-tools/pull/341/commits/06fbbe450e1f29e029b4f077d2da6c4a1e1bb9d8